tests/unittest/disaggregated/test_cache_reuse_adapter.py

```
@@ -650,6 +651,101 @@ def test_chunked_slice_entirely_stale_for_swa(self):
+    def test_destination_prefix_start_trims_source(self):
+        src, dst = self._prepare(
+            [10, 11, 12],
+            [20, 21, 22],
+            dst_start_token=8,
+        )
+
+        np.testing.assert_array_equal(src, [11, 12])
+        np.testing.assert_array_equal(dst, [20, 21])
+
+    def test_swa_requires_prompt_length(self):
+        with pytest.raises(ValueError, match="requires session.prompt_len"):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)